eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -125,6 +125,13 @@ public async Task<bool> InstallDotnetToolsAsync()
         {
             try
             {
+                await ProcessUtil.RunAsync($"{Options.DotnetRoot}/dotnet",
```

Comment:
nit: May be worth a comment stating we install `dotnet-dump` first to enable dumps when adding or listing NuGet sources and installing other tools (as well as during test execution).